src/modals/CreatingStudyModal.js

```diff
@@ -0,0 +1,351 @@
+      closeIcon
+      size="small"
+    >
+      <Modal.Header>External Study Resources</Modal.Header>
+      {error ? (
+        <Message
+          negative
+          icon="warning circle"
+          header="Error"
+          content={error.message}
+        />
+      ) : (
+        <Modal.Content>
+          {studyStatus === 'Creating' && (
+            <>
+              <Loader inline="centered" active size="massive" />
+              <Header textAlign="center" as="h2" icon>
+                Initializing Study Resources
+                <Header.Subheader>
+                  We're setting up your new study resources, sit tight.
+                </Header.Subheader>
+              </Header>
+            </>
+          )}
+          {studyStatus === 'Failed' && (
+            <Header textAlign="center" as="h2" icon className="noBefore">
+              <Icon name="warning sign" color="orange" />
+              Error Creating Study Resouces
+              <Header.Subheader>
+                Some study resources failed to initialize, please contact{' '}
+                <a
+                  href={`mailto:support@kidsfirstdrc.org?subject=Kids First Data Tracker Help&body=Hello, I had issue creating my study ${studyName}.`}
+                >
+                  support@kidsfirstdrc.org
+                </a>
+              </Header.Subheader>
+            </Header>
+          )}
+          {studyStatus === 'Created' && (
+            <Header textAlign="center" as="h2" icon className="noBefore">
+              <Icon name="check" color="green" />
+              Study Setup Completed
+              <Header.Subheader>
+                Your study resources were successfully created
+              </Header.Subheader>
+            </Header>
+          )}
+          <div className="text-wrap-75 margin-x-auto mt-30 width-fit-content">
+            <List>
+              <List.Item>
+                <List.Icon name="check" color="green" className="noPadding" />
+                <List.Content>
+                  <List.Header>
+                    Registered in Data Service: <code>{studyKfId}</code>
+                  </List.Header>
+                </List.Content>
+              </List.Item>
+
+              <List.Item>
+                {statusComponent[bucketStatus]}
+                <List.Content>
+                  <List.Header
+                    className={bucketStatus === 'Pending' ? 'text-grey' : ''}
+                  >
+                    {bucketStatus} storage bucket
+                  </List.Header>
+                </List.Content>
+                {studyByKfId && studyByKfId.bucket && (
+                  <List className="ml-15">
+                    <List.Item>
+                      <Icon name="aws" />
+                      <List.Content>
+                        <code>{studyByKfId.bucket}</code>
+                      </List.Content>
+                    </List.Item>
+                  </List>
+                )}
+              </List.Item>
+
```

Comment:
not necessary but could break each list item into its own component in a subdirectory just to make the CreateStudyModal component file shorted and easier to parse